elements/pl-string-input/pl-string-input.mustache

```diff
@@ -65,6 +65,7 @@
 
 {{#format}}
 <p>
-Your answer must be a string. Any symbolic expressions or numbers will be intepreted as a string.
+Your answer must be a text. Any symbolic expressions or numbers will be intepreted as a text (string type).
```

Comment:
In these two sentences, I think you mean just "text" and not "a text"? (Perhaps "plain text"?) Also, I would delete ` (string type)`.

Reply:
done